.travis.yml

```diff
@@ -3,10 +3,5 @@ language: node_js
 node_js:
   - 4
 
-before_script:
-  - npm i addons-linter -g
-
 script:
-  - npm run lint
   - npm run package
-  - addons-linter dist/addon.xpi
```

Comment:
Added **addons-linter** as a devDependency so this works locally too. Also moved the `lint` and `addons-linter` steps as npm hooks for the `package` script so running `$ npm run package` will automatically pre-lint and post-addons-linter as expected. \*magic\*